src/components/Toolbar.js

```diff
@@ -215,10 +218,11 @@ export default class Toolbar extends Component {
   }
 
   cancelEntity() {
-    this.props.editor && this.props.editor.focus();
     this.setState({
       editingEntity: null,
       error: null
+    }, () => {
+      this.props.draft && this.props.draft.focus();
```

Comment:
It is necessary to focus the draft element instead of the editor element. I opted to use the `draft` ref as a prop (similar to how `editor` is used) but I did also consider passing an `onCancelEntity` function from the `MegadraftEditor` component instead so it could be encapsulated.